compiler/llvm/clangUtil.cpp

```diff
@@ -3854,6 +3878,19 @@ void makeBinaryLLVM(void) {
     clangLDArgs = runtimeArgs;
   }
 
+  // Grab extra dependencies for multilocale libraries if needed.
+  if (fMultiLocaleInterop) {
+
+    // TODO: Has to be a more friendly way to do this?
+    clangLDArgs.push_back("-L/usr/local/lib");
+
+    std::string cmd = std::string(CHPL_HOME);
+    cmd += "/util/config/compileline --multilocale-lib-deps";
+    std::string libs = runCommand(cmd);
+    libs.erase(libs.size() - 1);
```

Comment:
What's going on with this erase call?

Reply:
Erasing trailing newline, will add comment.